docs/concept/wal.md

```diff
@@ -0,0 +1,56 @@
+# Wal Format
+
+![](/assets/wal-format.jpg)
+
+Wal file consist of a series of WalEntry，WalEntry contain as follows:
```

Comment:
Does A WAL is a single file? The canonical path is to generate a series of files that are named _segments_. Rotation will delete or overwrite a single segment instead of the whole WAL.